extensions/NV/SPV_NV_displacement_micromap.asciidoc

```diff
@@ -0,0 +1,297 @@
+SPV_NV_displacement_micromap
+============================
+
+Name Strings
+------------
+
+SPV_NV_displacement_micromap
+
+Contact
+-------
+
+See *Issues* list in the Khronos SPIRV-Registry repository:
+https://github.com/KhronosGroup/SPIRV-Registry
+
+Contributors
+------------
+
+- Eric Werness, NVIDIA
+- Ashwin Lele, NVIDIA
+- Pyarelal Knowles, NVIDIA
+- Christoph Kubisch, NVIDIA
+
+Status
+------
+
+- Provisional
+
+Version
+-------
+
+[width="40%",cols="25,25"]
+|========================================
+| Last Modified Date | 2023-08-01
+| Revision           | 1
+|========================================
+
+Dependencies
+------------
+
+This extension is written against the SPIR-V Specification,
+Version 1.1 Revision 4.
```

Comment:
This doesn't _need_ to be updated, but the SPIR-V 1.1 spec is pretty old.  Would it make sense to baseline this extension against the SPIR-V 1.6 spec instead?

Reply:
Good point, thanks! This should require 1.4 anyway, to match
[SPV_KHR_ray_tracing](https://github.com/KhronosGroup/SPIRV-Registry/blob/main/extensions/KHR/SPV_KHR_ray_tracing.asciidoc). Written-against might not matter as much but would be good to at least match ray tracing.